src/libstd/io/util.rs

```diff
@@ -64,27 +66,6 @@ impl BufRead for Empty {
     fn consume(&mut self, _n: usize) {}
 }
 
-/// A reader which infinitely yields one byte.
-#[stable(feature = "rust1", since = "1.0.0")]
-pub struct Repeat { byte: u8 }
-
-/// Creates an instance of a reader that infinitely repeats one byte.
-///
-/// All reads from this reader will succeed by filling the specified buffer with
-/// the given byte.
-#[stable(feature = "rust1", since = "1.0.0")]
-pub fn repeat(byte: u8) -> Repeat { Repeat { byte: byte } }
-
-#[stable(feature = "rust1", since = "1.0.0")]
-impl Read for Repeat {
-    fn read(&mut self, buf: &mut [u8]) -> io::Result<usize> {
-        for slot in &mut *buf {
-            *slot = self.byte;
-        }
-        Ok(buf.len())
-    }
-}
```

Comment:
I don't think this needs to move, you can access it through `std::io` instead of `std::io::utils`